src/views/auth/ComposeInternalLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import InternalLayout from '../../layout/InternalLayout'
+
+/**
+ * this is an HOC that renders the Internal Layout components of the application
+ * passing the required components as a props to the child.
+ * @function
+ * @param {object}  component - dashboard component.
+ * @return {HTMLElement}
+ */
+
+const ComposeInternalLayouts = (Component) => (passThroughProps) => (
+    <>
+        <InternalLayout {...passThroughProps} Component={Component} />
+    </>
+)
+
+export default ComposeInternalLayouts
```

Comment:
what is this file for? i don't think it should be here

Reply:
Yeah I dont know how this got committed let me remove it.